### DISCUSSION TITLE
Move .dotnet dir away from $HOME

### DISCUSSION THREAD

**Discussion by tony-sol** [dotnet/runtime#74465] [Q&A] [answered]
Is there any way to move .dotnet directory from $HOME on macos? Any env variable maybe or hidden XDG support?

**Comment by gfoidl** [comment]
You can install dotnet via zip/tar to any directory you want, just set the `DOTNET_ROOT` environment variable accordingly.
See [Download and manually install](https://docs.microsoft.com/en-us/dotnet/core/install/macos#download-and-manually-install) for more info.

**Reply by tony-sol** [reply]
`DOTNET_ROOT` environment variable is set
That is what i mean:
<img width="792" alt="Снимок экрана 2022-08-25 в 14 31 51" src="https://user-images.githubusercontent.com/33671815/186653335-5dffc603-6aa1-43cd-89dd-924adea7c44a.png">

**Comment by wzchua** [comment, marked as answer]
You can set `DOTNET_CLI_HOME`

**Reply by tony-sol** [reply]
Yeah, that helps
Thank you!
